spec/features/create_new_work_spec.rb

```diff
@@ -55,6 +56,10 @@
       fill_in 'Title of deposit', with: 'My Title'
       fill_in 'Contact email', with: user.email
 
+      # Adds second contact email
+      click_button '+ Add another email'
+      all('div.plain-container > div > input')[1].set(second_email)
```

Comment:
You should try to avoid using selectors in capybara.  There should be a label or aria-label that allows us to use `fill_in`.

Reply:
See comment above.

Reply:
Can we use something like `aria-label="<%= n.ordinalize %> contact"`?

Reply:
@jcoyne can you pair with @jermnelson to help finish this off?

Reply:
Leaving for now as `TEMPLATE_RECORD` along with `new Date().valueOf()` is still used for setting the ID of input in the `nested_form_controller.js`.